cmd/openshift-tests/images.go

```diff
@@ -38,6 +38,12 @@ import (
 // defaultTestImageMirrorLocation is where all Kube test inputs are sourced.
 const defaultTestImageMirrorLocation = "quay.io/openshift/community-e2e-images"
 
+// exceptions is a list of images we don't mirror temporarily due to various
+// problems. This list should ideally be empty.
+var exceptions = sets.NewString(
```

Comment:
Move this to the test/util/image package, not here

Reply:
Moved in the last push.